$OpenBSD: patch-configure_ac,v 1.7 2011/04/16 05:51:52 ajacoutot Exp $

XXX remove -DGTK_DISABLE_DEPRECATED to fix build with gtk>=2.24

Bump libgdata dependency to 0.7.0
The minimum dependency in GNOME is now 0.8.0, so we can bump the e-d-s
dependency to 0.7.0 and remove the various #ifdefs. See: bgo#566441
From upstream git: 69b8607f2dd6d399306905e011a6565319190066

--- configure.ac.orig	Mon Dec 27 14:19:33 2010
+++ configure.ac	Fri Apr 15 16:06:57 2011
@@ -36,7 +36,7 @@ m4_define([glib_minimum_version], [2.25.12])
 m4_define([gconf_minimum_version], [2.0.0])		dnl XXX Just a Guess
 m4_define([libxml_minimum_version], [2.0.0])		dnl XXX Just a Guess
 m4_define([libsoup_minimum_version], [2.3.0])
-m4_define([libgdata_minimum_version], [0.6.3])
+m4_define([libgdata_minimum_version], [0.7.0])
 m4_define([gnome_keyring_minimum_version], [2.20.1])
 m4_define([sqlite_minimum_version], [3.5])
 m4_define([libical_minimum_version], [0.43])
@@ -167,7 +167,6 @@ AS_COMPILER_FLAGS(WARNING_FLAGS,
 	"-DG_DISABLE_DEPRECATED
 	-DPANGO_DISABLE_DEPRECATED
 	-DGDK_PIXBUF_DISABLE_DEPRECATED
-	-DGTK_DISABLE_DEPRECATED
 	-DG_DISABLE_SINGLE_INCLUDES
 	-DGTK_DISABLE_SINGLE_INCLUDES
 	-DGSEAL_ENABLE
@@ -199,8 +198,6 @@ dnl	-Wstrict-aliasing=2
 AM_CPPFLAGS="$WARNING_FLAGS -fno-strict-aliasing"
 AC_SUBST(AM_CPPFLAGS)
 
-LDFLAGS="$LDFLAGS -Wl,--no-undefined"
-
 dnl ******************************
 dnl Initialize maintainer mode
 dnl ******************************
@@ -285,6 +282,13 @@ case "$host" in
 	AC_CACHE_VAL(ac_cv_have_addrinfo, [ac_cv_have_addrinfo=yes])
 	AC_DEFINE(_WIN32_WINNT, 0x501, [To get getaddrinfo etc declarations])
 	;;
+*openbsd*)
+	os_win32='no'
+	NO_UNDEFINED=''
+	SOCKET_LIBS=''
+	DL_LIB=''
+	LIBEXECDIR_IN_SERVER_FILE="$libexecdir"
+	;;
 *)
 	os_win32='no'
 	NO_UNDEFINED=''
@@ -324,12 +328,6 @@ PKG_CHECK_MODULES(GNOME_PLATFORM,
 	libxml-2.0 >= libxml_minimum_version
 	libsoup-2.4 >= libsoup_minimum_version
 	libgdata >= libgdata_minimum_version])
-
-dnl libgdata 0.7 will break API.  We're not going to deal with
-dnl that on a stable branch, so cap the supported version.
-if $PKG_CONFIG --atleast-version=0.7 libgdata; then
-	AC_MSG_ERROR([libgdata 0.7 is not supported, use 0.6 instead])
-fi
 
 LIBICAL_REQUIRED=libical_minimum_version
 AC_SUBST(LIBICAL_REQUIRED)
